dependencies.py

```diff
@@ -58,6 +58,10 @@ def extra_requirements() -> dict[str, list[str]]:
         "google-cloud": [
             "google-cloud-storage==2.14.0",
             "google-cloud-bigquery[pandas, bqstorage]==3.14.1",
+            # google-cloud-bigquery[bqstorage] requires google-cloud-bigquery-storage >= 2.6.0, but older (<2.17)
+            # versions of google-cloud-bigquery-storage runs into an issue
+            # (https://github.com/pypa/setuptools/issues/4476) with setuptools (ML-7273)
+            "google-cloud-bigquery-storage~=2.17",
```

Comment:
`~=2.17` is equivalent to `>=2.17,<3`.

Reply:
You're right, it's not necessary, because `google-cloud-bigquery-storage<3` is already a requirement of `google-cloud-bigquery[bqstorage]`. So it doesn't really matter either way except for purposes of readability.